content/lighthouse/configure-lighthouse.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Configure Lighthouse"
+date: 2020-09-23T15:29:49-04:00
+weight: 1
+---
+
+How to configure Lighthouse accessibility, SEO, and web-development best-practices audits against your Tugboat Previews
+
+- [Simple Lighthouse configuration](#simple-lighthouse-configuration)
+- [Use a custom Lighthouse configuration](#use-a-custom-lighthouse-configuration)
+- [Specify Lighthouse settings per-URL](#specify-lighthouse-settings-per-url)
+- [Group Lighthouse reports via service alias](#group-lighthouse-reports-via-service-alias)
+
+When Tugboat completes Google Lighthouse audits of your specified URLs, you'll be able to view the Lighthouse reports
+right in the Tugboat Dashboard. For more info about what you'll see, check out:
+[View Lighthouse Reports](../view-lighthouse-reports/).
+
+{{% notice tier %}} This feature is only available to [Tugboat Enterprise](https://www.tugboat.qa/enterprise) or
+On-Premise subscribers. If you'd like to run Lighthouse audits against your Tugboat Previews,
+[contact us](https://www.tugboat.qa/contact/) to update your project to an Enterprise subscription.{{% /notice %}}
+
+If you want to trigger Lighthouse audits for specific URLs, but turn visual diffs off for those urls, see:
+[Turn off visual diffs](/visual-diffs/configure-visual-diffs/#turn-off-visual-diffs).
+
+## Simple Lighthouse configuration
+
+To configure which pages get Lighthouse accessibility, SEO, and best practices audits, specify the _relative URLs_ of
+the pages in a `urls` key in the [service definition](/setting-up-services/). In this format, each URL can be a string,
+and you can list multiple urls.
+
+```yaml
+services:
+  apache:
+    urls:
+      # Conduct Lighthouse audits of the these URLs using the default options
+      - /
+      - /blog
+      - /about
+```
+
+## Use a custom Lighthouse configuration
+
+Tugboat uses the default Lighthouse configuration, but disables some server performance metrics. If you'd like to use a
+custom Lighthouse configuration, to tweak audit scoring or add custom checks, you can pass a custom Lighthouse
+configuration object to use when generating reports.
+
+You can find documentation for creating a custom Lighthouse config at:
+[GoogleChrome/lighthouse blob on GitHub](https://github.com/GoogleChrome/lighthouse/blob/HEAD/docs/configuration.md).
```

Comment:
It's an SEO/usability/accessibility best practice to use relevant keywords in anchor text instead of using naked URLs. I've restructured this sentence to try to make it parse better - let me know if you like this version more.